.github/workflows/main.yml

```
@@ -58,10 +58,13 @@ jobs:
         include:
           - os: ubuntu-latest
             pyodide_packages: "tag:core,numpy${{ needs.test-scipy-trigger.outputs.test-scipy == 'true' && ',scipy' || '' }}"
+            # Temporary for this PR: remove this when merging
+          - os: ubuntu-latest
+            pyodide_packages: "tag:core,numpy${{ needs.test-matplotlib-trigger.outputs.test-matplotlib == 'true' && ',matplotlib' || '' }}"
           - os: macos-14
             pyodide_packages: "tag:core,numpy"
     runs-on: ${{ matrix.os }}
-    needs: [test-scipy-trigger]
+    needs: [test-scipy-trigger, test-matplotlib-trigger]
```

```suggestion
    needs: [test-scipy-trigger]
```

(Reverting my changes)